bootstrap.inc.sh

```diff
@@ -132,7 +121,10 @@ function _bootstrap_configure_common() {
     _bootstrap_download "_common/$common_file" "$BOOTSTRAP_COMMON/$common_file"
   done
 
-  # _bootstrap_download_directory "_common/assets/sil-logos-2024" "$BOOTSTRAP_COMMON/assets/sil-logos-2024"
+  for img_file in "${IMG_FILES[@]}"; do
+    mkdir -p "$BOOTSTRAP_COMMON/assets/sil-logos-2024/"
+    _bootstrap_download "_common/assets/sil-logos-2024/$img_file" "$BOOTSTRAP_COMMON/assets/sil-logos-2024/$img_file"
```

Comment:
Can we use a var for the assets folder? (After setting `BOOTSTRAP_COMMON` in l. 95)
something like
```bash
  local ASSET_LOGOS="assets/sil-logos-2024"
```
Then these could use that var like
```suggestion
    mkdir -p "$BOOTSTRAP_COMMON/$ASSET_LOGOS"
    _bootstrap_download "_common/$ASSET_LOGOS/$img_file" "$BOOTSTRAP_COMMON/$ASSET_LOGOS/$img_file"
```

Reply:
Thansks @darcywong00.